src/client/components/Symbol/Checkmark.tsx

```diff
@@ -1,47 +1,42 @@
-import React, { FunctionComponent } from 'react';
+import React, { FC } from 'react';
 import styled from 'styled-components';
+import CSS from 'csstype';
 import STRINGS from '../../assets/strings.json';
 
-interface Props {
+interface Props extends Pick<CSS.Properties, 'color' | 'height' | 'width'> {
```

Comment:
`Pick<>` type is cool

Reply:
It provides really nice comments/descriptions on all of the pick'ed fields in this case. Really cool stuff.